movie-browser/src/view/CardContainer.js

```diff
@@ -1,13 +1,17 @@
 /* eslint-disable no-unused-vars */
 import React from 'react'
-import Card from './view/Card'
+import Card from './Card'
+
+const NoOfMinMovies = 6
```

Comment:
what does `Min` stand for? Unclear.

Reply:
Changed into "NoOfMinimumMovies". I think I might need to change the name totally at the next stage, but I'll think about it later.